wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ViewModel/ConversationListViewControllerViewModel.swift

```diff
@@ -117,6 +113,20 @@ extension ConversationListViewController.ViewModel {
         }
 
         updateObserverTokensForActiveTeam()
+
+        notificationCenter.addObserver(
+            self,
+            selector: #selector(handleApplicationDidBecomeActiveNotification(_:)),
+            name: UIApplication.didBecomeActiveNotification,
+            object: nil
+        )
+
+        notificationCenter.addObserver(
+            self,
+            selector: #selector(updateE2EICertificateStatus),
+            name: .e2eiCertificateChanged,
+            object: nil
+        )
```

Comment:
Shouldn't we use the block based variant:
`addObserver(forName:object:queue:using:)`

This way we can avoid the selector `@objc` syntax and go to Swift only.